ci/workflows/main-cron.yml

```diff
@@ -367,7 +367,27 @@ steps:
     timeout_in_minutes: 80
     retry: *auto-retry
 
-  - label: "recovery test (madsim)"
+  # sql backend recovery tests
+  - label: "recovery test (sql,madsim)"
+    key: "recovery-test-deterministic-sql"
+    command: "TEST_NUM=12 KILL_RATE=1.0 BACKGROUND_DDL_RATE=0.0 USE_SQL_BACKEND=true timeout 65m ci/scripts/deterministic-recovery-test.sh"
+    # NOTE(kwannoel): It will only run when the recovery tests label is added currently.
+    # This is because there are currently some bugs which cause the test to fail.
+    if: |
+      build.pull_request.labels includes "ci/run-recovery-test-deterministic-simulation"
```

Comment:
Shall we use a separate label for SQL madsim?

Reply:
Fixed.